cep-XXXX.md

```diff
@@ -0,0 +1,265 @@
+
+#### `./info/hash_input.json`
+
+Required.
+
+A free-form dictionary that maps `str` keys to `str` values. The dictionary MAY be empty.
+
+Its contents MAY be used to generate a hash string that uniquely identifies the package.
+
+#### `./info/about.json`
+
+Optional.
+
+A dictionary that maps `str` keys to arbitrary values.
+
+The contents of this file usually include the `about` section of the originating recipe for the package, plus metadata about the context of the build process. The following keys SHOULD be included and, if present, they MUST conform to the proposed schema:
+
+- `channels: list[str]`. Conda channels used to solve the build and test environments.
+- `description: str`. A longer form description of the package.
+- `dev_url: str`. URL pointing to the development website (often a repository) of the package.
+- `doc_url: str`. URL pointing to the documentation website of the package.
+- `env_vars: dict[str, str]`. Allow-listed environment variables set during the package build.
+- `extra: dict[str, Any]`: A free-form dictionary of arbitrary metadata. This MAY be used to record provenance metadata as described in [CEP PR#113](https://github.com/conda/ceps/pull/113).
+- `home: str`. URL pointing to the homepage of the package.
+- `license: str`. SPDX license identifier for the package.
+- `summary: str`: A short summary of the package (usually one sentence).
+
+These keys are commonly found, but are now considered deprecated:
+
+- `license_file: str | list[str]`: Paths to the license files, relative to the `recipe/` directory.
+- `license_family: str`. Use `license` instead.
+- `tags: list[str]`.
+- `identifiers: list[str]`.
+- `keywords: list[str]`.
+
+Additional keys MAY be allowed. Some examples include vendor information such as the conda-build and conda versions used at build time, along with the packages present in the conda `base` environment (`conda_build_version`, `conda_version`, `root_pkgs`, respectively).
+
+#### `./info/link.json`
+
+Optional.
+
+A dictionary containing important metadata for installation (linking) operations. Its contents MUST conform to the following schema:
+
+- `package_metadata_version: int`. Version of this file. The only known value is `1`.
+- `noarch: dict[str, Any]`. A dictionary describing the `noarch` configuration of the package. Its keys MUST conform to:
+  - `type: Literal["generic", "python"]`. Type of `noarch` package.
+  - `entry_points: list[str]`. If `type == "python"`, this key MUST list the console entry points of the Python package, as a list of strings with syntax `{executable-name}={dotted-import-path}:{function-name}` (for example `bsdiff4 = bsdiff4.cli:main_bsdiff4`).
+- `preferred_env: dict[str, str]`. Deprecated, optional. It MUST have two keys: `name: str` (name of the preferred environment) and `executable_paths: list[str]` (list of preferred executable paths in that environment).
```

Comment:
For consistency with other sections, lets keep the list of deprecated keys seperate 

```suggestion
These keys are commonly found, but are now considered deprecated:

- `preferred_env: dict[str, str]`. It MUST have two keys: `name: str` (name of the preferred environment) and `executable_paths: list[str]` (list of preferred executable paths in that environment).
```